docker-compose.yml

```diff
@@ -96,6 +96,12 @@ services:
     environment:
       - MICRO_REGISTRY=mdns
       - DB_HOST=datastore:27017
+      - OS_AUTH_AUTHSTRATEGY=keystone
```

Comment:
Why is this required?

Reply:
This is required to call our identity service to retrieve the SK for GCP credentials decryption

Reply:
Ok